commit_data.py

```diff
@@ -11,60 +11,46 @@
 
 presidents = [
     President(first_name="Uhuru", last_name="Kenyatta",
-              location="Kenya", votes=0),
+              location="Kenya", votes=0,party="Jubilee"),
     President(first_name="Raila", last_name="Odinga",
-              location="Kenya", votes=0),
+              location="Kenya", votes=0, party="ODM"),
+    
+]
+
+deputy-presidents = [
```

Comment:
Deputies uses the Deputy class. A deputy instance should also have a president_id in it. 
```suggestion
deputies = [ Deputy(first_name="William", last_name="Ruto", location="Kenya", president_id=1, votes=0)]
```
You can use this as a template.